docroot/sites/now.uiowa.edu/modules/now_core/now_core.module

```diff
@@ -85,3 +138,61 @@ function now_core_node_presave($entity) {
     );
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function now_core_form_taxonomy_term_confirm_form_alter(&$form, FormStateInterface $form_state) {
+  // Check/prevent are auto-tags from being deleted on single delete.
+  // Only need to alter the delete operation form.
+  if ($form_state->getFormObject()->getOperation() !== 'delete') {
+    return;
+  }
+  $tag = $form_state
+    ->getFormObject()
+    ->getEntity();
+
+  if (in_array($tag->id(), [
```

Comment:
Sorry to nitpick but can we use _article_type_tag_map() to get this list of ids so we only maintain the list in one place?

Reply:
Absolutely--pushed up a slight refactor for this.